scripts/extract_definitions.py

```diff
@@ -1,30 +1,14 @@
 import numpy as np
-import tensorflow as tf
 import tqdm
 import typer
-from transformers import AutoTokenizer
-from transformers import TFDistilBertForSequenceClassification
 
 from mars.db import collections
 from mars.db.db_fields import SENTENCE, IS_DEFINITION
+from mars.definition_model import DistilBertBaseUncased
 
 
-def predict_single_sentence(sentence: str, model, tokenizer: AutoTokenizer) -> float:
-    inputs = tokenizer(sentence, return_tensors="tf")
-    inputs["labels"] = tf.reshape(tf.constant(1), (-1, 1))  # Batch size 1
-    outputs = model(inputs)
-    predictions = tf.math.softmax(outputs.logits, axis=-1)
-    return np.array(predictions)[0][1]
-
-
-def main(TRANSFORMER):
-    tokenizer = AutoTokenizer.from_pretrained(TRANSFORMER)
-
-    if TRANSFORMER == "distilbert-base-uncased":
-        model = TFDistilBertForSequenceClassification.from_pretrained("../models/" + TRANSFORMER)
-
-    else:
-        raise Exception
+def main():
```

Comment:
Ten skrypt nie jest nam narazie potrzebny, ale skoro już jest, to dobrze, żeby uruchamiał tylko funkcję document_definition_scoring. Analogicznie do https://github.com/ModelOriented/AI-strategies-papers-regulations-monitoring/blob/dev/scripts/segment_pdf_and_html.py

Reply:
👍